LDAR_Sim/src/parameters/high_level_parameters.py

```diff
@@ -19,7 +19,8 @@
 #
 # ------------------------------------------------------------------------------
 
-from typing import Any, override
+from typing import Any
+from typing_extensions import override
```

Comment:
Why was this change necessary?

Reply:
It's not using the correct import, and was failing to run.